### DISCUSSION TITLE
Image change when promoted

### DISCUSSION THREAD

**Discussion by amr-el-sayed** [Phobos-developers/Phobos#483] [Ideas and feedback]
Similar to the unit weapons, units/buildings appearances can change when they get promoted.
For example:
Image=GI
VeteranImage=GI_V
EliteImage=GI_E

**Comment by Thrifinesma** [comment]
Ares has already implemented conversion via promotion, I don't think there is any necessity to do such a already possible feature.

**Reply by amr-el-sayed** [reply]
Ares apply it for units but not buildings.
